core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1124,12 +1184,24 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/WorkspaceMigrationMetadata'
+        400:
+          description: Some or all workspaces in billing project were ineligible for migration.
```

Comment:
Same comment as above-- you cannot call this method on a billing project where some of the workspaces are not in US multi-region? I would just expect it to ignore those workspaces instead of erroring, as otherwise the method doesn't seem very useful.